WebSTR/templates/about.html

```diff
@@ -79,6 +79,50 @@ <h2>Programmatic access - WebSTR API</h2>
 			<p>
 				<h2>Available datasets</h2>
 			</p>
+
+
+                        <table>
+    <tr>
+      <th>Panel alias</th>
+      <th>Human Genome Assembly Version</th>
+      <th>Annotation Method</th>
+      <th>Genotyping Methods</th>
+      <th>STRs</th>
+      <th>Cohorts</th>
+      <th>Samples</th>
+      <th>Available Data</th>
+    </tr>
+    <tr>
+      <td>ensembletr_hg38</td>
+      <td>GRCh38.p2</td>
+      <td>TRF</td>
+      <td>EnsembleTR<br>ExpansionHunter<br>GangSTR<br>HipSTR</td>
+      <td>1,710,833 total reference panel with allele frequency data available<br>35,998<br>1,133,225<br>1,331,280</td>
+      <td>1000GenomesH3Africa</td>
```

Comment:
need space or "/" between 1000Genomes and H3Africa (these are two different cohorts)